lisa/tools/iperf3.py

```diff
@@ -251,6 +252,19 @@ def run_as_client(
             expected_exit_code_failure_message="fail to lanuch iperf3 client",
         )
 
+    def wait_start_client(self, process: Process, timeout: int = 300) -> bool:
+        # IPerf output emits lines of the following form when it is running
+        # 132.00-133.00 sec   167 MBytes  1.40 Gbits/sec    5    626 KBytes
+        # check if stdout buffers contain "bits/sec" to determine if running
+        start_time = time.time()
+        while time.time() - start_time < timeout:
+            if "bits/sec" in process._stdout_writer._buffer:
+                self._log.debug(process._stdout_writer._buffer)
+                return True
+            time.sleep(0.1)
```

Comment:
If it takes more than 10 seconds, sleep 1 second is ok. There may be perf impact if check it too frequently.